provider/datagen/src/transform/cldr/units/mod.rs

```diff
@@ -62,7 +167,55 @@ fn test_basic() {
         .unwrap();
 
     let constants = &und.get().to_owned().constants_map;
+    let ft_to_m = constants.get("ft_to_m").unwrap();
+    let expected_ft_to_m =
+        GenericFraction::<BigUint>::new(BigUint::from(3048u32), BigUint::from(10000u32));
+    assert_eq!(
+        ft_to_m,
+        zerovec::ule::encode_varule_to_box(&ConstantValue {
+            numerator: expected_ft_to_m.numer().unwrap().to_le_bytes().into(),
+            denominator: expected_ft_to_m.denom().unwrap().to_le_bytes().into(),
+            sign: Sign::Positive,
+            constant_type: ConstantExactness::Actual,
+        })
+        .as_ref()
+    );
+
+    let ft2_to_m2 = constants.get("ft2_to_m2").unwrap();
+    let expected_ft2_to_m2 = GenericFraction::<BigUint>::new(
+        BigUint::from(3048u32).pow(2),
+        BigUint::from(10000u32).pow(2),
+    );
```

Comment:
This can be

```suggestion
    let expected_ft2_to_m2 = expected_ft_to_m.pow(2);
```

if you use `BigRational`.

Reply:
At least move the `pow` outside the fraction?

Reply:
done